demo/admin/src/products/categories/ProductsSelectGrid.tsx

```diff
@@ -0,0 +1,226 @@
+import { useQuery } from "@apollo/client";
+import {
+    DataGridToolbar,
+    GridCellContent,
+    GridColDef,
+    GridColumnsButton,
+    GridFilterButton,
+    muiGridFilterToGql,
+    muiGridSortToGql,
+    ToolbarFillSpace,
+    ToolbarItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import { useTheme } from "@mui/material";
+import { GridCallbackDetails } from "@mui/x-data-grid/models/api";
+import { GridInputSelectionModel, GridSelectionModel } from "@mui/x-data-grid/models/gridSelectionModel";
+import { DataGridPro, GridFilterInputSingleSelect, GridToolbarQuickFilter } from "@mui/x-data-grid-pro";
+import gql from "graphql-tag";
```

Comment:
Please import this from `@apollo/client`

Reply:
I think the import doesn't work (at least on my device)

It should probably be `import { gql, useQuery } from "@apollo/client";` instead of `import gql, { useQuery } from "@apollo/client";`